src/test/java/dev/cdevents/constants/CDEventTypesTest.java

```diff
@@ -0,0 +1,15 @@
+package dev.cdevents.constants;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import dev.cdevents.constants.CDEventConstants.CDEventTypes;
+
+public class CDEventTypesTest {
+	@Test
+	void allEventTypeNamesMatchExpectedPattern() {
+		assertThat(CDEventTypes.values()).extracting(CDEventTypes::getEventType)
+				.allMatch(type -> type.matches("[a-z\\.]+\\.\\d+\\.\\d+\\.\\d+"));
```

Comment:
NIT: This could even be `dev\\.cdevents\\.[a-z]+\\.[a-z]+\\.\\d+\\.\\d+\\.\\d+"`

Reply:
Done.